.drone.yml

```diff
@@ -17,6 +17,18 @@ pipeline:
     # dont clone submodules
     recursive: false
 
+  display-status:
+    image: harbor.ci.drone.local/library/vic-integration-test:1.36
```

Comment:
You may want to use some other light weight image for this.

Reply:
I considered using some other image, but almost the entire pipeline seems to use this one. Homogeneity seems valuable, and the whole `display-status` step takes less than a second of a 10+ minute process so I'm not sure what using something lighter weight would gain us.

Reply:
That timestamp probably doesn't include time to download that image which is `1.95GB` size. It doesn't matter much in passing case but mostly in failure case. Even though we are not gaining much in terms of time savings I think its always better to use lightest possible image ( docker best practice :-) )

Reply:
that image is hosted locally on a gigabit network and cached.  We almost never download the full image on a build.

Reply:
The entire drone build takes ~9 seconds when this check fails (see, e.g., https://ci.vcna.io/vmware/vic/14467 or https://ci.vcna.io/vmware/vic/14468). Reducing that isn't worth the added complexity of introducing a different image into the process.

Reply:
>  I think its always better to use lightest possible image ( docker best practice :-) )

My understanding is that this is a best practice because it reduces complexity; fewer things in an image mean fewer things that can go wrong.

In this case, using a different image _increases_ complexity in a different way.